CHANGES.md

```diff
@@ -72,6 +72,12 @@ OpenSSL 3.5
 
    *Paul Dale*
 
+ * Extended `OPENSSL_ia32cap` support to accommodate additional `CPUID`
```

Comment:
This well-written summary can be further shortened to replace the PR title and/or the first line of the commit message.

Reply:
Thank you, updated the commit message